packages/storybook8/stories/Composites/ChatComposite/snippets/Utils.tsx

```diff
@@ -37,31 +37,39 @@ const sendMessagesAsBot = async (
   messages: string[]
 ): Promise<void> => {
   const chatClient = new ChatClient(envUrl, new AzureCommunicationTokenCredential(token));
-  const threadClient = await chatClient.getChatThreadClient(threadId);
+  const threadClient = chatClient.getChatThreadClient(threadId);
```

Comment:
Any reason that we don't need to await for this function?

Reply:
It's not an async function, await was redundant here